ee/LICENSE

```diff
@@ -1,9 +1,7 @@
-The following license is used by Contember Enterprise Edition available for download from: https://contember.com/
+The Contember Enterprise Edition (EE) license (the “EE License”) Copyright (c) 2018 - present manGoweb, s.r.o.
 
-The Contember Enterprise Edition (EE) license (the “EE License”) Copyright (c) 2018 - present manGoweb, spol. s r.o.
+The features of the Contember Enterprise Edition software and associated documentation files (the “Software”) may only be used if you (and any entity that you represent) have a valid Contember Enterprise Edition subscription for the Contember Enterprise Edition that you are running, accessing, or using. You may, however, utilize the free version of the Software (with several features not enabled) under this license without a license key or subscription provided that you otherwise comply with the terms and conditions of this Agreement. Subject to the foregoing, except as explicitly permitted in the EE Terms, it is forbidden to copy, merge, modify, publish, distribute, sublicense, stream, perform, display, create derivative works of and/or sell the Software in either source or executable form without written agreement from manGoweb, s.r.o.. Notwithstanding anything to the contrary, free versions of the Software are provided “AS-IS” without indemnification, support, or warranties of any kind, expressed or implied. You assume all risk associated with any use of free versions of the Software.
```

Comment:
```suggestion
The features of the Contember Enterprise Edition software and associated documentation files (the “Software”) may only be used if you (and any entity that you represent) have a valid Contember Enterprise Edition subscription for the Contember Enterprise Edition that you are running, accessing, or using. You may, however, utilize the free version of the Software (with several features not enabled) under this license without a license key or subscription provided that you otherwise comply with the terms and conditions of this Agreement. Subject to the foregoing, except as explicitly permitted in the EE Terms, it is forbidden to copy, merge, modify, publish, distribute, sublicense, stream, perform, display, create derivative works of and/or sell the Software in either source or executable form without written agreement from manGoweb, s.r.o. Notwithstanding anything to the contrary, free versions of the Software are provided “AS-IS” without indemnification, support, or warranties of any kind, expressed or implied. You assume all risk associated with any use of free versions of the Software.
```